hazelcast/src/main/java/com/hazelcast/nio/tcp/TcpIpConnectionManager.java

```diff
@@ -78,9 +79,9 @@ public TcpIpConnectionMonitor createNew(Address endpoint) {
 
     private final boolean socketNoDelay;
 
-    private final ConcurrentMap<Address, Connection> connectionsMap = new ConcurrentHashMap<Address, Connection>(100);
+    private final ConcurrentHashMap<Address, Connection> connectionsMap = new ConcurrentHashMap<Address, Connection>(100);
```

Comment:
I would just supress the issue in checkstyle for the time being. We need to cleanup the io package anyway.